Support 2 digit outlet numbers

ehemmete opened the pull request:
Changed the substring cut off on the index value for 2 digit outlets.  This project now works with APC AP8965 and APDU9965 which have 24 outlets per PDU.

ehemmete committed:
Update index.php

Changed the substring cut off on the index value for 2 digit outlets.

disisto commented:
Thank you very much for submitting the pull request. I can only speak for the devices AP7920, AP7921, and AP7920B, as I only own these.

When I apply the proposed change, I encounter an HTTP 500 error when switching, which according to the logs looks like this:

```BASH
AH01071: Got error 'PHP message: PHP Warning:  snmp3_walk(): Invalid object identifier: 1.3.6.1.4.1.318.1.1.12.3.5.1.1.4. 8 in /var/www/html/apc/index.php on line 75; PHP message: PHP Fatal error:  Uncaught TypeError: implode(): Argument #2 ($array) must be of type ?array, bool given in /var/www/html/apc/index.php:78\nStack trace:\n#0 /var/www/html/apc/index.php(78): implode()\n#1 {main}\n  thrown in /var/www/html/apc/index.php on line 78', referer: https://xxxx/
```

In `snmp3_walk()`, the socket (here #8) is being transmitted with a space, which causes the error.

I have rewritten the affected line as follows:

``` BASH
$index = trim(substr($value['outlet'], -2));
```

Could you check if this works in your case? Thank you in advance.

ehemmete committed:
Update index.php

Added trim to the index command, to handle PDUs with single digit outlet numbers and two digit outlet numbers.

ehemmete commented:
Yes, I can confirm that adding the trim keeps it working on our PDUs.  I've updated the pull request to include that.

disisto committed:
Improved: Bootstrap version upgraded to 5.3.3
Improved: Code has been improved to support PDUs with two-digit outlets. <a href="https://github.com/disisto/apc-switched-rack-pdu-control-panel/pull/5/commits">PR #5</a>

⚠️ This is the last iteration of the 1.x versions.

With a focus on maintainability, usability, performance, and security, the principle of the monolithic/portable script is being abandoned. Version 2.x, which has been completely rebuilt from the ground up and incorporates feedback from the past 12 months, will be released shortly.

If you have not yet integrated the script into your environment, it is advisable to wait with the adjustments.

disisto commented:
Awesome. Thank you very much.
